django/template/__init__.py

```diff
@@ -64,5 +64,8 @@
 # Library management
 from .library import Library                                            # NOQA isort:skip
 
+# Autoreloader implementation
+from . import autoreload                                                # NOQA isort:skip
```

Comment:
This is definitely not the best way to invoke this module, but I really don't know how else to do it. This file is a kind of entrypoint to `django.template` submodules, so we don't want to pollute it if we can avoid it.

Reply:
So question, can the autoreloader take care of these registrations itself? (Haven't looked into it but...)

Reply:
Part of the improvements I made to the autoreloader was to decouple it from registrations like this. The only one we have right now, prior to this, is the translation cache clearing.

However I didn't consider modules like this, where the module scope is effectively the API and adding any imports to it might cause issues down the line. I'd _really_ like to avoid adding anything to the autoreloader specifically for this, I'd honestly prefer to do:

```python
# Import the .autoreload module to trigger the registrations of signals, but delete 
# the reference afterwards so that it is not exposed.
from . import autoreload
del autoreload
```

Reply:
On consideration, I'd take the first bit of that...

```
# Import the .autoreload module to trigger the registrations of signals.
```